04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+\subsection{Example Distribution}
+\subsubsection{Example of the Distribution for Foraging Return (Deer Hunting - Payoff Dominant)}
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/images/Distribution of Foraging returns Deer Hunting.PNG}
+    \caption{Deer hunting payoff dominant Distribution for foraging return}
+    \label{fig:Distribution of Foraging returns Deer Hunting}
+\end{figure}
+
+To increase the risk of deer hunting, a Bernoulli Random Variable ($D$) was used to prevent guaranteed resource returns after a hunt. While the Exponential Decay ($W$) is to disincentive too much resources being placed in hunting and hoping for a high return. However, this all comes with the benefit of significantly higher ($2\times$) returns than fishing.
+
+Figure~\ref{fig:Distribution of Foraging returns Deer Hunting} represents the return utility which will be multiplied by an output scale to give the return payoff. The Figure~\ref{fig:Distribution of Foraging returns Deer Hunting} shows the average \textbf{expected mean} return utility, in colour, after $1000$ iterations which is close to the \textbf{real mean}.
+
+\newpage
+\subsubsection{Fish hunting - Risk Aversion}
+
+Fishing is similar to Deer hunting, except that it uses only a Normal Distribution ($F$), which focuses on avoiding risk by being very predictable and safe. Figure~\ref{fig:Distribution of Foraging returns Fishing} illustrates the return utility for fishing.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/images/Distribution of Foraging returns Fishing.PNG}
+    \caption{Fishing Risk aversion Distribution for foraging return}
+    \label{fig:Distribution of Foraging returns Fishing}
+\end{figure}
+
+As seen in Figure~\ref{fig:Distribution of Foraging returns Deer Hunting}and Figure~\ref{fig:Distribution of Foraging returns Fishing}, the return for the deer is almost double that of the fish return whilst the fish return is much easier to obtain due to the lower cost of catching fish.
+
+\newpage
+\subsection{Solution Concepts}
+\subsubsection{Dominant Strategy}
+
+In this implementation, there is no dominant strategy. Therefore, if an agent decides to go deer hunting, whereas all the other islands are fishing, that agent would not get the best return possible. Hence it is not the best strategy. Whilst if an agent goes fishing and there are a sufficient amount of agents deer hunting, then that agent would be passing on the opportunity of to gain a larger return in deer hunting. Hence, it is not the best strategy either.
```

Comment:
on the second one it works - "Hence, this is not the best strategy either." is better though because it's clearer what the this/it is referring to